gcsa/serializers/event_serializer.py

```diff
@@ -27,6 +27,7 @@ def _to_json(cls, event: Event):
             'description': event.description,
             'location': event.location,
             'recurrence': event.recurrence,
+            'id': event.event_id,
```

Comment:
Good catch. That's actually a bug. Thanks for noting that.

But could you please move it to the top of the dictionary (seems kinda logical), above the `'summary'`?

Reply:
done